### ISSUE TITLE
Webcam Horizontal Flip

### ISSUE THREAD

**Issue by shady-azouz** [justadudewhohacks/face-api.js#386]
I'm using Face Api to detect facial expressions through the webcam.

I'm trying to flip the webcam's video horizontally, so that the person sees himself mirrored using this code in the css file for the video and the canvas:

```
-o-transform: scaleX(-1);
-moz-transform: scaleX(-1);
-webkit-transform: scaleX(-1);
-ms-transform: scaleX(-1);
transform: scaleX(-1);
```

The issue is, the text is also flipped. I was looking for a way to flip the text initially, so that when it's mirrored, it would be in the correct orientation.

Or is there a better approach than mine?

**Comment by justadudewhohacks** [comment]
Why are you flipping the overlay canvas? Simply flip the video and draw onto the unflipped canvas. You just have to flip the x and y coordinates of the text position then.

**Comment by jingzhi93** [comment]
hi @justadudewhohacks , Im facing the same issue as well. May I know how to get the x and y coordinates of the text position?

**Comment by leonardloh** [comment]
hi @justadudewhohacks , I'm facing the same issue as well. May I know how to get the x and y position of the text?

**Comment by LUGAMAFE** [comment]
![116749758_756949811795251_4864592742820408411_n](https://user-images.githubusercontent.com/24960025/88740298-669e1f80-d102-11ea-954c-d3bcd2442df9.png)
Bro same problem how da hell i flip the x and y coordinates of the text position?

**Comment by jammind** [comment]
It was straightforward to mirror the coordinates in the raw detection result by 'faceapi.detectAllFaces'.
![image](https://user-images.githubusercontent.com/2723305/209502564-1a1d9b6d-ac32-4cfd-8a2c-28ebe925a7cc.png)

But after a chain of async calls like 'await rawDetections.withFaceLandmarks().withFaceExpressions().withFaceDescriptors()', the coordinates are no longer absolute and hard to understand. Neither can I mirror the raw detection result since I have to use await to get it but then it can't be chained to get further landmarks.

```Javascript
    const rawDetections = faceapi.detectAllFaces(video, new faceapi.TinyFaceDetectorOptions()) // Raw detection result. Can't see any useful data by inspection.
    const rawLandmarks = await rawDetections // Meaningful coordinates. Easy to mirror. Cannot be chained to get further landmarks.
    const detections = await rawDetections.withFaceLandmarks().withFaceExpressions().withFaceDescriptors() // Works only with raw detection result.
```

Drawing the video frames on canvas and mirroring them using canvas context shall work. But it will hurt the performance a lot. The best practice is to translate the coordinates in your raw detection algorithm.

In short, could you provide an option to mirror the raw input in 'faceapi.detectAllFaces'?
